todoman/formatters.py

```diff
@@ -82,7 +82,6 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
-
```

Comment:
An empty line shouldn't make a test fail (definitely not in this context). Are you sure you're not mistaken with something else?